src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../types/movie';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" data-cy="card" key={movie.imdbId}>
```

Comment:
use `ul` and `li` instead + don't forget to add `key`

Reply:
in this task we don't need use ul/li